### ISSUE TITLE
[Shaders] Shadow Shaders

### ISSUE THREAD

**Issue by eassym0ney** [sp614x/optifine#5953]
When I am playing normally with the BLS shaders, Sildur suits me well, but when it comes to using serious not everything is bad, the problem is that when I enter either my house or anywhere the items look a bit dark, not exaggerated but if it is annoying and weird that they look like this and I have configured everything, even my graphics card is a GTX 1650 and I don't know how to solve it, maybe it is the Optifine.

## Steps to Reproduce
1. Normally entering a house or closed place looks very dark even with lighting.
2. Lighting torches and lighting remnants inside places.

## OptiFine Version
OptiFine HD U G9 pre24

## Installation Method
Put the JAR file inside the mods folder

## Fabric/Forge Version
Fabric: fabric-loader-0.11.6-1.17

## OptiFabric Version
optifabric-1.11.9

In summary with the other I have no problems
![2021-07-01_13 28 22](https://user-images.githubusercontent.com/86752087/124167147-b6533c80-da71-11eb-9672-644be44a6a2f.png)

**Comment by Whitelisted1** [comment]
This looks similar to #5895, nobody has found a solution yet

**Comment by KaiAF** [comment]
Already addressed here: #5895
